js/base/functions/generic.js

```diff
@@ -31,6 +31,7 @@ module.exports =
     , index
     , ordered: x => x // a stub to keep assoc keys in order (in JS it does nothing, it's mostly for Python)
     , unique:  x => Array.from (index (x))
+    , arrayConcat: (a, b) => a.concat (b)
```

Comment:
I think we need to keep it, since it's used in too many exchanges )

Reply:
it is kept i just moved it to a different folder